clients/drcachesim/tests/cache_replacement_policy_unit_test.cpp

```diff
@@ -35,6 +35,7 @@
 #undef NDEBUG
 #include <assert.h>
 #include "cache_replacement_policy_unit_test.h"
+#include "simulator/cache_fifo.h"
 #include "simulator/cache_lru.h"
 
 class cache_lru_test_t : public cache_lru_t {
```

Comment:
The idea is to remove this and use the template.

Reply:
Done.